Casks/t/tuxera-ntfs.rb

```diff
@@ -8,8 +8,8 @@
   homepage "https://ntfsformac.tuxera.com/"
 
   livecheck do
-    url "https://ntfsformac.tuxera.com/support/"
-    regex(/Release\s*?v?(\d+(?:\.\d+)*)/i)
+    url "https://download.tuxera.com/mac/tuxerantfs_latest.dmg/"
```

Comment:
```suggestion
    url "https://download.tuxera.com/mac/tuxerantfs_latest.dmg"
```

It doesn’t seem to matter in this instance, but the website doesn’t add a trailing slash to the download URL.